HHS/Head-Start-TTADP
# [TTAHUB-1688] Allow editing of FEI goals on the RTR

## Description of change
Several needed improvements to the conditional prompts/FEI goal feature. Most pertinently, the ability to edit an existing curated goal's prompts has been added to the RTR's edit goal form.

## How to test
- Create a multi-**recipient** report. Select the FEI goal, and approve the report. You should be unable to select FEI root causes for the goal, since it is a multi-recipient report.
- Create a second report, using one of the two previous recipients. Update the goal's FEI root cause, submit, and approve the report.
- Go to the RTR and the goals and objectives tab for the first recipient. You should see the FEI goal, and when you edit it, you'll see the root cause as read only.
- Go to the RTR and the goals and objectives tab for the second recipient. Since you've never added root causes via an activity report, you should be able to edit the root causes on this recipient's FEI goal. Ensure you can do so without issue.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1688

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
